src/main/kotlin/racingcar/controller/InputValidator.kt

```diff
@@ -0,0 +1,46 @@
+package racingcar.controller
+
+object CarNameValidator {
+    const val ERROR_EMPTY_NAME = "자동차 이름은 공백이거나 빈 문자열일 수 없습니다."
+    const val ERROR_LENGTH_EXCEEDED = "자동차 이름은 5자 이하여야 합니다."
+    const val ERROR_CONTAINS_SPACE = "자동차 이름은 공백을 포함할 수 없습니다."
+    const val ERROR_DUPLICATE_NAME = "자동차 이름은 중복될 수 없습니다."
+}
+
+object RaceCountValidator {
+    const val ERROR_EMPTY_COUNT = "경주 시도 횟수는 빈 문자열일 수 없습니다."
+    const val ERROR_NOT_NUMBER = "경주 시도 횟수는 숫자여야 합니다."
+    const val ERROR_INVALID_COUNT = "경주 시도 횟수는 1 이상이어야 합니다."
+}
+
+object InputValidator {
+    
+    fun validateCarName(names: List<String>) {
+        require(names.all { it.isNotBlank() }) {
```

Comment:
names.all 을 사용하여 모든 이름에 대해 무결한지 파악하고 있습니다.
하나라도 흠이 있다면 예외를 도출할것입니다.
names.any 를 사용하여 하나라도 흠이 있는지 판별한다면 모든 이름에 대해 판별하는 것과 비교해 같은기능을 하면서 조금 더 러닝타임을 줄일수 있는 코드가 될것같아요
 ```suggestion
        require(names.any { it.isBlank() }) {
            CarNameValidator.ERROR_EMPTY_NAME
        }
        require(names.any { it.length > 5 }) {
            CarNameValidator.ERROR_LENGTH_EXCEEDED
        }
        require(names.any { it.contains(" ") }) {
            CarNameValidator.ERROR_CONTAINS_SPACE
        }
```

Reply:
음 all의 경우도 만족하지 않으면 바로 탈출하는 걸로 알고 있습니다.

Reply:
좋은 의견 감사합니다. 하지만,
"모든 항목이 공백이 아니어야 한다"는 의미를 직관적으로 전달하기 위해서는 all으로 작성하는 것이 더 좋다고 생각합니다!

Reply:
저도 any 쪽에 표를 남깁니다.

저도 all이 바로 탈출하는지 몰랐기 때문에 생긴 문제이긴 합니다만,
"모든 항목이 공백이 아니어야 한다"는 의미는 모든 항목을 돌린다고 생각이 들어요. 
표현 자체는 "어떤 항목이라도 공백일 수 없다"가 예외처리에서 직관적이라고 생각합니다.

Reply:
지금보니 밑에 코드를 any로 다 했네요! 통일성 측면에서도 any로 다음엔 해야겠습니다 감사합니다